### PR TITLE
Topic recognition - UNet3D Implementation on CSIRO Male Pelvic dataset

### PR DESCRIPTION
**Pull request by shravanshivpuri**
# Introduction
This project implements the UNet3D[[1]](#ref1) Convolutional Neural Network architecture in Tensorflow 2.6, and applies it on the (downsampled) CSIRO 3D Male Pelvic MRI dataset[[2]](#ref2) to achieve dense volumetric segmentation of the pelvis.

The model achieved the following Dice Similarity Coefficients on the **test set**. This result was attained after training for ~13 hours on a 20GB partition of an Nvidia Tesla A100 Tensor Core GPU.

|Class|Label|DSC|
|--|--|--|
|0|Background|0.998|
|1|Body|0.987|
|2|Bones|0.926|
|3|Bladder|0.920|
|4|Rectum|0.858|
|5|Prostate|0.799|

Refer to the <a href="https://github.com/shravanshivpuri/PatternFlow/blob/topic-recognition/recognition/SS_Unet3D/README.md">README.md</a> for more details.

### CODE REVIEW

**Comment by wooboyeong** [comment]
README
* Please show example outputs in README
* Don't plot loss and Dice on the same plot - should be plotted separately (plot train loss and val loss on the same plot instead)

General Comments
* Code OK for NORMAL
* Commit messages OK
* Pull request OK

**Comment by shravanshivpuri** [comment]
Feedback incorporated.

**Comment by shakes76** [comment]
## TF/Torch Usage
Adequate use and implementation
## Good Practice (Design/Commenting)
Good spacing and comments
No Header blocks -1
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present 
no Data leakage 
Difficulty: Normal -5
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch)
No feedback required
Request Description good
